website/docs/reference/actions.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Actions
+---
+
+:::info Availability
+
+Unleash actions is an **enterprise-only** feature released as beta in Unleash v5.11.
+
+:::
+## Overview
+
+Actions enable our users to execute automated actions based on signals received from an external system.
+
+![Actions executed by Unleash.](/img/signals/action-execution.png)
+
+## Creating actions
+
+:::info Permissions
+
+During the beta phase, creating actions require admin permissions.
+
+:::
+
+Actions are scoped to a project. To be able to create actions, you need to navigate to the project settings and find the **Actions** tab.
+
+![Actions tab.](/img/signals/actions-tab.png)
+
+### Name and description
+When creating a new action you first need to specify a name and a description. This will help you identify your action and give it a meaningful description.
+
+### Source and filters
+
+Then you need to select a source. If you haven't created a signal endpoint yet, you can do so by clicking the **Create new signal endpoint** button (more information on this in the [Signals](./signals.mdx) section).
+
+Once selected the source, you'll be able to see a preview of the latest signal which will help you to define different filters that need to match before executing the action.
```

Comment:
```suggestion
If the source you selected already received at least one signal, you'll be able to see a preview of the latest signal received. This can help you define the different filters that need to match for the action to be executed.
```